Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -409,6 +405,15 @@ - (void)invalidate
     });
 }
 
+IntPoint ScrollerMac::lastKnownMousePositionInScrollbar() const
+{
+    // When we dont have an update from the Web Process, return
```

Comment:
This comment is only relevant to UI-side compositng, but you have avoid breaking non-UI-side composting too!

Reply:
ScrollerPairMac/ScrollerMac are meant to be UI-side compositing only classes. This code is handled by the delegate in ScrollbarsContollerMac when in non-UI-side composting mode.